File: acme.cashflow.domain/DTO/Account/FluxoDeCaixa.cs
using acme.cashflow.domain.DTO.Util;
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.cashflow.domain.DTO.Account
{
    public class FluxoDeCaixa : AbstractEntity
    {
        public FluxoDeCaixa()
        {
            FluxoDeCaixasReceitas = new HashSet<FluxoDeCaixaReceita>();
            FluxoDeCaixasDespesas = new HashSet<FluxoDeCaixaDespesa>();
        }

        public decimal SaldoOperacional { get; set; }
        public decimal SaldoFinalCaixa { get; set; }
        public decimal SaldoInicial { get; set; }
        public Guid CompetenciaId { get; set; }

        public virtual Competencia Competencia { get; set; }
        public virtual ICollection<FluxoDeCaixaReceita> FluxoDeCaixasReceitas { get; set; }
        public virtual ICollection<FluxoDeCaixaDespesa> FluxoDeCaixasDespesas { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/fluxo-de-caixa.ts
import { FluxoDeCaixaReceita } from './fluxo-de-caixa-receita';
import { AbstractEntityCompetencia } from "../abstract-entity-competencia";
import { FluxoDeCaixaDespesa } from "./fluxo-de-caixa-despesa";

export class FluxoDeCaixa extends AbstractEntityCompetencia{
  public  saldoOperacional:number;
  public  saldoFinalCaixa :number;
  public  saldoInicial :number;

  public fluxoDeCaixasReceitas: FluxoDeCaixaReceita[];
  public fluxoDeCaixasDespesas: FluxoDeCaixaDespesa[];
}
